### DISCUSSION TITLE
CPU Frequency formatting

### DISCUSSION THREAD

**Discussion by HalJordan** [brndnmtthws/conky#1270] [Ideas]
Is there any easy way to keep that from jumping back and forth when it goes from 3 to 4 digits when using MHz?

**Comment by Caellian** [comment]
Sadly no currently, I'm transferring this to ideas category.

It would be cool to add a general variable for formatting command output such as:
```
${format <format string> <inner variable> [inner variable args]*}
```

This would require typed variable outputs or adding support for registering format handlers for all of them (and default to string).
